dotnet/Microsoft.MacCatalyst.Templates/maccatalyst/AppDelegate.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Foundation;
+using UIKit;
+
+namespace MacCatalystApp1
+{
+    [Register("AppDelegate")]
+    public class AppDelegate : UIApplicationDelegate
+    {
+        public override UIWindow Window
+        {
+            get;
+            set;
+        }
+
+        public override bool FinishedLaunching(UIApplication application, NSDictionary launchOptions)
+        {
+            // create a new window instance based on the screen size
+            Window = new UIWindow(UIScreen.MainScreen.Bounds);
```

Comment:
source code format is inconsistent, e.g. space before `(`

Reply:
Should I `dotnet format` every template to use default VS settings?

Or should I `dotnet format` using the `.editorconfig` in xamarin-macios?

Reply:
as much as I'd like to use our (mono) style it's likely best to follow VS[4m] style used for other templates